rock_gazebo.orogen

```diff
@@ -64,7 +70,9 @@ task_context "ModelTask" do
     input_port "joints_cmd", "/base/samples/Joints"
     output_port "joints_samples", "/base/samples/Joints"
     # Link export
+    dynamic_input_port /.*/, '/base/samples/Wrench'
```

Comment:
If we do go for the naming scheme as I suggest below, then update these declarations to match the scheme

For instance,

```
dynamic_input_port /_wrench$/, '/base/samples/Wrench'
```